lib/mixins/annotations.js

```diff
@@ -129,6 +129,26 @@ export default {
     return this.annotate(x, y, w, h, options);
   },
 
+  fileAttachmentAnnotation(x, y, w, h, file = {}, options = {}) {
```

Comment:
```suggestion
  fileAnnotation(x, y, w, h, file = {}, options = {}) {
```
I would simplify the name since to embed a file uses only `file`. I keeps consistency with other api (text -> textAnnotation)

Reply:
Great idea, fixed!